httpconnection.go

```diff
@@ -126,7 +127,14 @@ func NewHTTPConnection(ctx context.Context, address string, options ...func(*htt
 	var conn Connection
 	switch {
 	case negotiateResponse.hasTransport("WebTransports"):
-		// TODO
```

Comment:
This way the way I noticed btw as this lead to a nasty nil pointer at a completely unrelated place (connection nil and error nil at the same time).